flink-connectors/flink-connector-jdbc/src/test/resources/org/apache/flink/connector/jdbc/table/JdbcTablePlanTest.xml

```diff
@@ -51,4 +51,22 @@ TableSourceScan(table=[[default_catalog, default_database, jdbc, project=[decima
 ]]>
     </Resource>
   </TestCase>
+  <TestCase name="testFilterPushdown">
+	<Resource name="sql">
+		<![CDATA[SELECT id, time_col, real_col FROM jdbc WHERE id = 900001 AND time_col <> '11:11:11.000111' OR double_col >= -1000.23]]>
+	</Resource>
+	<Resource name="ast">
+		<![CDATA[
+LogicalProject(id=[$0], time_col=[$3], real_col=[$4])
++- LogicalFilter(condition=[OR(AND(=($0, 900001), <>($3, _UTF-16LE'11:11:11.000111')), >=($5, -1000.23:DECIMAL(6, 2)))])
+   +- LogicalTableScan(table=[[default_catalog, default_database, jdbc]])
+]]>
+	</Resource>
+	<Resource name="optimized exec plan">
+		<![CDATA[
+Calc(select=[id, time_col, real_col], where=[((time_col <> '11:11:11.000111') OR (double_col >= -1000.23))])
```

Comment:
why the condition still exists in the Calc?

Reply:
I figure out, its because the column type Time wasn't supported, I've added support here: https://github.com/apache/flink/pull/20140/commits/74fe5eed158482920a9ec36ac8a7aad549ee94fa

Thanks for calling it out.